tests/h/groups/__init___test.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from h.groups import includeme
+from h.groups.search import GroupAuthFilter
+
+
+def test_includeme_registers_search_filter(pyramid_config):
+    includeme(pyramid_config)
+    assert GroupAuthFilter in pyramid_config.memex_get_search_filters()
+
+
+@pytest.fixture
+def pyramid_config(pyramid_config):
+    pyramid_config.add_directive('memex_set_groupfinder', lambda c, f: None)
+
+    filters = []
+    pyramid_config.add_directive('memex_add_search_filter',
```

Comment:
It would have required too much knowledge about memex, because memex tried to initialise the search index which failed when you don't have ES running on localhost:9200